LinkedList.cpp

```diff
@@ -0,0 +1,160 @@
+#include "ListInterface.h"
+
+/*
+<Author>: Tucker J. Polomik github.com/TheNergaL
+<Name of algorithm>: Singly Linked-list
+<Brief description>:

```

Comment:
- Remove the angle brackets
- Put a newline after the ending `:`
- Indent the comment block by a single level

Like so:
```C++
/*
    Author:
    Tucker J. Polomik (https://github.com/TheNergaL)

    Single linked list:
    <brief description of a single linked list goes here>

    Time complexity:
    Element access: O(N)
    Insertion: O(1)
    Deletion: O(1)
    where N is the number of nodes (elements) in the list.

    Space complexity:
    O(N), where N is the number of nodes (elements) in the list.
*/
```